src/foam/nanos/auth/CapabilityAuthService.js

```diff
@@ -175,9 +191,23 @@ foam.CLASS({
         implied by a capability in userCapabilityJunctions for a given user.
       `,
       javaCode: `
-        if ( x == null || permission == null ) return false;
-        if ( x.get(Session.class) == null ) return false;
-        if ( user == null || ! user.getEnabled() ) return false;
+        if ( x == null || permission == null ) {
+          StdoutLogger.instance().debug(getClass().getSimpleName() + ".capabilityCheck",
```

Comment:
Do we need the '+'? If we just use a comma instead does it just concatenate the arguments or separate them with a comma? The reason I ask is that using a ',' would avoid creating a StringBuilder and a new String.